common/components/SubTabs/index.tsx

```diff
@@ -1,63 +0,0 @@
-import React from 'react';
-import classnames from 'classnames';
-import { TranslateType } from 'translations';
-import { Link } from 'react-router-dom';
-import './SubTabs.scss';
-
-export interface Tab<SubTabProps = null> {
-  path: string;
-  name: TranslateType;
-  isDisabled?(props: SubTabProps | {}): boolean;
-  render(props: SubTabProps | {}): React.ReactElement<any> | null;
-}
-
-export interface Props<SubTabProps = null> {
-  activeTab?: string;
-  root: string;
-  tabs: Tab<SubTabProps>[];
-  sideBar?: React.ReactElement<any> | React.Component | React.StatelessComponent;
-  subTabProps?: SubTabProps;
-  onTabChange?(): void;
-}
-
-interface State {
-  isOpenModal: boolean;
-}
-
-export default class SubTabs<SubTabProps = null> extends React.Component<
-  Props<SubTabProps>,
-  State
-> {
-  public render() {
-    const { tabs, sideBar } = this.props;
-    const activeTab = this.props.activeTab || tabs[0].path;
-    const tab = tabs.find(t => t.path === activeTab) || tabs[0];
-    const columnSize = sideBar ? 8 : 12;
-
-    return (
-      <div className="SubTabs row">
-        <div className={`SubTabs-tabs col-sm-${columnSize}`}>
-          {tabs.map(t => (
-            <Link
-              className={classnames({
-                'SubTabs-tabs-link': true,
-                'is-active': t.path === activeTab,
-                'is-disabled': t.isDisabled && t.isDisabled(this.props.subTabProps || {})
-              })}
-              to={`/${this.props.root}/${t.path}`}
-              key={t.path}
-              onClick={this.props.onTabChange}
-            >
-              {t.name}
-            </Link>
-          ))}
-        </div>
-
-        <main className={`SubTabs-content col-sm-${columnSize}`} key={tab.path}>
-          {tab.render(this.props.subTabProps || {})}
-        </main>
-        {this.props.sideBar ? this.props.sideBar : null}
-      </div>
-    );
-  }
-}
```

Comment:
I don't like the idea of removing the subtabs component, I think we were considering transitioning signing and contract pages to it instead of the oversized header things.

Reply:
Yeah, we definitely want to keep this. I'm going to rely on it for disabling features that are only valid on specific networks as well.

Reply:
I don't think the subtabs component is necessary. Everything that it can do can be done much cleaner by using [NavLink](https://reacttraining.com/react-router/web/api/NavLink)

example using [NavLink](https://reacttraining.com/react-router/web/api/NavLink)
```
const tabs = [
  {
    path: 'valid',
    name: 'foo'
  },
  {
    path: 'invalid',
    name: 'bar'
  }
];

const validTabs = ['foo']

<div className="SubTabs row">
	<div className="SubTabs-tabs col-sm-8">
		{tabs.map((t, i) => (
			<NavLink
				className={`SubTabs-tabs-link ${!validTabs.includes(t.name) ? 'is-disabled' : ''}`}
				activeClassName="is-active"
				to={currentPath + '/' + t.path}
				key={i}
			>
				{t.name}
			</NavLink>
		))}
	</div>
</div>
```